packages/components/order-tracking/README.md

```diff
@@ -0,0 +1,42 @@
+# Order tracking
+
+## Install
+
+```sh
+yarn add @ovh-ux/order-tracking
+```
+## Usage
+
+```js
+import angular from 'angular';
+import {
+  module as orderTrackingModule,
+  component as orderTrackingComponent
+} from '@ovh-ux/order-tracking';
+
+angular
+  .module('myApp', [
+    orderTrackingModule,
+    uiRouter,
+  ])
+  .config(($stateProvider) => {
+    $stateProvider.state('order-tracking', {
+      url: '/order-tracking/{orderId}',
+      component: orderTrackingComponent,
```

Comment:
I don't think this will work, `component` only accepts `string`s and your `component` exports an `object`

https://ui-router.github.io/ng1/docs/latest/interfaces/ng1.ng1statedeclaration.html#component